content/concepts/transport.md

```diff
@@ -115,3 +115,190 @@ and some places in the codebase still use the "swarm" terminology.
 {{% /notice %}}
 
 [definition_switch]: /reference/glossary/#switch
+
+## QUIC
+
+Transport protocols continue to improve transport methods in an attempt to alleviate 
+the shortcomings of the transport layer. 
+
+{{% notice "note" %}}
+
+Recalling the purpose of transports
```

Comment:
This paragraph is technically correct, but I don’t like the framing as transports being the thing that introduce reliability.

How about listing the problems that existed with TCP that led to the development of QUIC, namely:
1. it’s a single byte stream exposed by the kernel, so streams layered on top of TCP experience HoL blocking (this would be a good place to explain HoLB)
2. ossification: because TCP is unencrypted, middleboxes have been reading (and modifying!) TCP header fields and might break in unexpected ways when they encounter anything they don’t like
3. the TCP 3-way handshake is inefficient, as it spends 1 RTT on verifying the client’s address (that’s what the 3-way handshake is there for)

Reply:
nice suggestion, thought it would be good to also better define the flow and create subsection, with this being under its own